doc/cephadm/osd.rst

```diff
@@ -211,6 +211,7 @@ If you want to avoid this behavior (disable automatic creation of OSD on availab
 
 * For cephadm, see also :ref:`cephadm-spec-unmanaged`.
 
+.. _cephadm-osd-removal:
```

Comment:
NIT, but I feel like this makes more sense at the bottom of the "Remove an OSD" section rather than right before it starts.

Reply:
all the other reference link things go before the section.